### PR TITLE
Events update

### PR DESCRIPTION
**Pull request by elisadinsmore**
Adding missing events to the events page

## Summary by CodeRabbit

- **Chores**
	- Updated `.gitignore` to include more directories for better repository cleanliness.

- **Documentation**
	- Added documentation for new document-related events, including creation, update, and deletion of documents, creation of sheet snapshots, completion of commits, updates to agents, and addition or removal of guests in spaces.

### CODE REVIEW

**Comment by elisadinsmore** [comment]
@ashleygmulligan2 this is also ready for a review!

**Review by ashleygmulligan2 — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 .idea
-.obsidian
\ No newline at end of file
+.obsidian
```
Comment (ashleygmulligan2) [inline .gitignore:2]: not sure what this is

File: package.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "dependencies": {
+    "mintlify": "^4.0.131",
+    "sharp": "^0.32.6"
```
Comment (ashleygmulligan2) [inline package.json:4]: prob don't wan tthis

**Review by ashleygmulligan2 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (elisadinsmore) [reply .gitignore:2]: I don't either, but it was already there. Or are you commenting on the node_modules line?

File: package.json (hunk 2 above)
Comment (ashleygmulligan2) [reply package.json]: can we get rid of this in this PR? @elisadinsmore

File: .gitignore (hunk 1 above)
Comment (ashleygmulligan2) [reply .gitignore:2]: can we get rid of this in this PR? @elisadinsmore

**Review by ashleygmulligan2 — APPROVED** [review]

**Pull request merged**